lib/algos/list/basic.js

```diff
@@ -44,6 +46,50 @@ class List extends Extension {
         return params;
     }
 
+    /**
+     * Filters customAttributes sub-object if present
+     *
+     * @param {String} value - The JSON value of a listing item
+     *
+     * @return {Boolean} Returns false if the value is filtered out, else true.
+     */
+    customFilter(value) {
+        let ret = false;
+        try {
+            const _value = JSON.parse(value);
+            if (_value.customAttributes !== undefined) {
+                const keys = Object.keys(_value.customAttributes);
+                let i;
+                for (i = 0; i < keys.length; i++) {
+                    const key = keys[i];
```

Comment:
You can use for... of here
```suggestion
                for (const key of Object.keys(_value.customAttributes)) {
```